evoting/lib/transaction.go

```diff
@@ -75,8 +77,29 @@ func NewTransaction(data interface{}, user uint32, signature []byte) *Transactio
 	return transaction
 }
 
+// Digest appends the digits of sciper to master genesis skipblock ID
+func (t *Transaction) Digest(roster *onet.Roster, genesis skipchain.SkipBlockID) []byte {
+	var election *Election
+	if t.Election != nil {
+		election = t.Election
+	} else {
+		election, _ = GetElection(roster, genesis)
+	}
+	// Master or Link transaction
+	if election == nil {
+		return nil
+	}
+	message := election.Master
+	for _, c := range strconv.Itoa(int(t.User)) {
+		d, _ := strconv.Atoi(string(c))
+		message = append(message, byte(d))
```

Comment:
Is it not possible to do `append(message, c)`?

Reply:
Would have to change the Javascript side too. This is not really ideal, but it works, and I think it is important to keep moving forward on this.